crates/web-sys/webidls/enabled/HTMLTableCellElement.webidl

```diff
@@ -21,6 +21,10 @@ interface HTMLTableCellElement : HTMLElement {
            [CEReactions, SetterThrows]
            attribute DOMString headers;
   readonly attribute long cellIndex;
+           [CEReactions, SetterThrows]
+           attribute DOMString abbr;
+           [CEReactions, SetterThrows]
+           attribute DOMString scope;
```

Comment:
Looking at the spec I couldn't find anything on these setters actually able to throw.
```suggestion
           [CEReactions]
           attribute DOMString abbr;
           [CEReactions]
           attribute DOMString scope;
```

Reply:
Done.